test/suites/example/docker-compose.override.yml

```diff
@@ -15,11 +15,38 @@ services:
       - ../../example/extra-install.sh:/extra-install.sh
     environment:
       - QS_PUBLIC_SCHEME_HOST_AND_PORT=http://quickstatements.svc:80
```

Comment:
NOTE: These override health checks are here so this can be released independently, but go away in a subsequent PR (the nginx proxy/ Quickstatements fix one) which applies them directly in the /example docker compose.